earthly/rust/Earthfile

```diff
@@ -19,7 +21,7 @@ rust-base:
     # The ACTUAL version of rust that will be used, and available targets
     # is controlled by a `rust-toolchain.toml` file when the `SETUP` UDC is run.
     # HOWEVER, It is enforced that the rust version in `rust-toolchain.toml` MUST match this version.
-    FROM rust:1.73-alpine3.18
+    FROM rust:1.74-alpine3.18
```

Comment:
Should we use 1.75 if we are bumping the rust version?

Reply:
Yea, will bump to the 1.75